src/Okta.Idx.Sdk/IdxClient.cs

```diff
@@ -237,6 +240,106 @@ internal async Task<IIdxResponse> IntrospectAsync(IIdxContext idxContext, Cancel
                 request, cancellationToken).ConfigureAwait(false);
         }
 
+        /// <inheritdoc/>
+        public async Task<WidgetSignInResponse> StartWidgetSignInAsync(CancellationToken cancellationToken = default)
+        {
+            IIdxContext idxContext = await this.InteractAsync();
+            return new WidgetSignInResponse
+            {
+                IdxContext = idxContext,
+                SignInWidgetConfiguration = new SignInWidgetConfiguration(this.Configuration, idxContext),
+            };
+        }
+
+        /// <inheritdoc/>
+        public async Task<OktaTokens> RedeemInteractionCodeAsync(IIdxContext idxContext, string interactionCode, Action<Exception> exceptionHandler = null, CancellationToken cancellationToken = default)
+        {
+            exceptionHandler = exceptionHandler ?? LogError;
+            try
+            {
+                Uri issuerUri = new Uri(Configuration.Issuer);
+                Uri tokenUri = new Uri(GetNormalizedUriString(issuerUri.ToString(), "v1/token"));
```

Comment:
What happens here if the developer passes an issuerUri without the trailing `/`.  Can we add a check in here to make sure it includes the `/` or are we ok with failing due to malformed URL?